tests/main/interfaces-locale-control/task.yaml

```diff
@@ -47,11 +47,11 @@ execute: |
     snap disconnect locale-control-consumer:locale-control ubuntu-core:locale-control
 
     echo "Then the snap is not able to read the locale configuration"
-    if su -l -c "locale-control-consumer.get LANG 2>${PWD}/locale-read.error" test; then
```

Comment:
I wonder the same thing. Doesn't make sense to me that $PWD wouldn't work here in Trusty.

Also, the restore needs to be fixed if that ends up going in. It's removing a file that doesn't exist, and leaving the one that exists behind.